jdaviz/configs/specviz/plugins/viewers.py

```diff
@@ -51,7 +54,7 @@ class SpecvizProfileView(BqplotProfileView, JdavizViewerMixin):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self._initialize_toolbar_nested()
+        self._initialize_toolbar_nested(default_tool_priority=self.tools_nested[4])
```

Comment:
for consistency with the way cubeviz behaves

Reply:
See https://github.com/spacetelescope/jdaviz/pull/1115/files#r826427496